consensus/misc/eip1559/eip1559_test.go

```diff
@@ -218,3 +220,96 @@ func TestCalcBaseFeeOptimismHolocene(t *testing.T) {
 		}
 	}
 }
+
+// TestCalcBaseFeeMinBaseFee assumes all blocks are jovian blocks that have the minimum base fee feature enabled.
+// It tests that the minimum base fee is enforced when the computed base fee is less than the minimum base fee.
+func TestCalcBaseFeeMinBaseFee(t *testing.T) {
+	parentGasLimit := uint64(30_000_000)
+	denom := uint64(50)
+	elasticity := uint64(3)
+
+	tests := []struct {
+		parentBaseFee   int64
+		parentGasUsed   uint64
+		expectedBaseFee uint64
+		minBaseFee      uint64
```

Comment:
Super nit: I would like the expectation to be the last item in the struct. Then the test cases are a bunch of input followed by an expectation.

Reply:
[fe136e1](https://github.com/ethereum-optimism/op-geth/pull/648/commits/fe136e1ce0c04e7419cb36036661e855d79f1c6c)